00_hello_world/main.go

```diff
@@ -8,6 +8,21 @@ import (
 
 var out io.Writer = os.Stdout
 
+func fib(i int) int {
+	var a = 0
+	var b = 1
+	fmt.Fprintln(out, b)
+	var sum int
```

Comment:
Although `sum` happens to default to zero, it's better to express the intent that it must start from zero: `sum := 0`.

Reply:
Done. Thanks.